frontend/simple/views/Join.vue

```diff
@@ -133,25 +133,33 @@ export default {
   methods: {
     accept: async function () {
       try {
-        // post acceptance event to the group contract
-        this.errorMsg = null
-        let latest = await backend.latestHash(this.$route.query.groupId)
-        let acceptance = new Events.HashableGroupAcceptInvitation(
-          {
-            username: this.$store.state.loggedIn.name,
-            identityContractId: this.$store.state.loggedIn.identityContractId,
-            inviteHash: this.$route.query.inviteHash,
-            acceptanceDate: new Date()
-          },
-          latest
-        )
-        // let the group know we've accepted their invite
-        await backend.publishLogEntry(this.$route.query.groupId, acceptance)
-        // sync the group's contract state
-        await this.$store.dispatch('syncContractWithServer', this.$route.query.groupId)
-        // after syncing, we can set the current group
         this.$store.commit('setCurrentGroupId', this.$route.query.groupId)
+        await backend.subscribe(this.$route.query.groupId)
```

Comment:
`backend.(un)subscribe` must never be called, here or anywhere else. Remove all instances.